pkg/kubelet/kubelet.go

```diff
@@ -600,12 +600,17 @@ func NewMainKubelet(
 	if err != nil {
 		return nil, err
 	}
-	whitelist, err := sysctl.NewWhitelist(sysctlWhitelist)
+	safeWhitelist, err := sysctl.NewWhitelist(sysctl.SafeSysctlWhitelist(), api.SysctlsPodAnnotationKey)
+	if err != nil {
+		return nil, err
+	}
+	unsafeWhitelist, err := sysctl.NewWhitelist(allowedUnsafeSysctls, api.UnsafeSysctlsPodAnnotationKey)
```

Comment:
We have to concatenate safe and unsafe here to allow promotion of sysctls onto the whitelist